mysite/templates/mysite/register.html

```diff
@@ -0,0 +1,11 @@
+{% extends "mysite/base_generic.html" %}
+
+{% block content %}
+
+<form method="POST">
+    {% csrf_token %} {{ form.as_p }}
+    <button type="submit">Submit</button>
+</form>
+
+{% endblock %}
+
```

Comment:
What is the difference between register.html and signup.html ? The name is confusing here.

Reply:
Signup.html was added by mistake. Deleted now.